res/css/structures/_RightPanel.pcss

```diff
@@ -83,7 +83,7 @@ limitations under the License.
 
     h2,
     p {
-        font-size: $font-14px;
+        font: var(--cpd-font-body-md-regular);
```

Comment:
I'm confused as to when we use react-sdk's `$font-X` variables and when to use `--cpd` stuff

Reply:
Everything should eventually use the Compound tokens.

I mapped the obvious things to a Compound alternative, and made the rest look roughly aligned based on the review that @janogarcia gave me.